README.md

```diff
@@ -2,51 +2,64 @@
 
 [![Build Status](https://travis-ci.org/andreypopp/esy-ocaml-project.svg?branch=master)](https://travis-ci.org/andreypopp/esy-ocaml-project)
 
-A project which demonstrates an OCaml workflow with esy.
+A project which demonstrates an OCaml workflow with `esy`.
 
-## Installation
-
-While we are in an experiment mode you need to use a custom registry for `@opam`
-scope (this is where we publish opam packages to):
+`esy` is an overlay on top of `package.json` based package managers, that
+enables `package.json` packages to use compilers and environments quickly, and
+easily.
 
-    % npm config set @opam:registry https://opam-npm.andreypopp.com
+## Installation
 
-Then just use `npm install` as usual:
+You can install the beta using `npm install -g`.
 
-    % npm install .
+    npm install -g git://github.com/jordwalke/esy.git#beta-v0.0.2
 
 **WARNING:** Only `npm` is supported right now (not `yarn`, `pnpm`, or `ied`).
 This restriction is temporary and will be lifted in the future.
 
 ## Build
 
-    % npm run build
+    % esy build
 
 ## Clean build artifcats
 
-    % npm run clean
+    % esy clean
 
 ## Shell into build environment
 
-    % npm run shell
+    % esy shell
 
 ## Run preconfigured OCaml REPL
 
-    % npm run repl
+    % esy repl
 
 ## Merlin
 
 Merlin integration is included. If your editor of choice has Merlin
 plugin/extension then you can run:
 
-    % npm run editor
+    % esy $EDITOR
```

Comment:
This is actually not quite true yet since `esy command` often scrubs the environment so clean that it won't find your editor in the path! But eventually I imagine this will become true.